braindecode/datasets/tuh.py

```diff
@@ -192,10 +200,11 @@ class TUHAbnormal(TUH):
         description.
     """
     def __init__(self, path, recording_ids=None, target_name='pathological',
-                 preload=False, add_physician_reports=False):
+                 preload=False, add_physician_reports=False, n_jobs=-1):
```

Comment:
```suggestion
                 preload=False, add_physician_reports=False, n_jobs=1):
```

use n_jobs=1 by default everywhere please. It can create crazy issues on shared machines otherwise

Reply:
Alright, thanks. Will use default of 1.